### DISCUSSION TITLE
Is there a shortcut to switch between displays?

### DISCUSSION THREAD

**Discussion by usuall** [rustdesk/rustdesk#10349] [Q&A]
RustDesk is great. I have one question. 
Is there a shortcut to quickly switch between multiple displays when using a remote PC? 
In the case of Google Desktop, it was very convenient as shortcut keys could be configured. 
Can anyone help me with this?

**Comment by xlionjuan** [comment]
![圖片](https://github.com/user-attachments/assets/bddea6f3-eac8-4dd8-8ab1-7258e34441eb)

**Reply by zacaj** [reply]
This shows buttons, not a keyboard shortcut

**Reply by xlionjuan** [reply]
> This shows buttons, not a keyboard shortcut

Button shortcut 😂
